atmo/settings.py

```diff
@@ -531,6 +509,20 @@ class Base(Core):
 
     LOGGING_USE_JSON = values.BooleanValue(False)
 
+    OIDC_RP_CLIENT_ID = values.SecretValue(
+        environ_name='OIDC_RP_CLIENT_ID', environ_prefix=None)
+    OIDC_RP_CLIENT_SECRET = values.SecretValue(
+        environ_name='OIDC_RP_CLIENT_SECRET', environ_prefix=None)
+    OIDC_OP_AUTHORIZATION_ENDPOINT = values.Value(
+        environ_name='OIDC_OP_AUTHORIZATION_ENDPOINT', environ_prefix=None)
+    OIDC_OP_TOKEN_ENDPOINT = values.Value(
+        environ_name='OIDC_OP_TOKEN_ENDPOINT', environ_prefix=None)
+    OIDC_OP_USER_ENDPOINT = values.Value(
+        environ_name='OIDC_OP_USER_ENDPOINT', environ_prefix=None)
+    OIDC_OP_DOMAIN = values.Value(
+        environ_name='OIDC_OP_DOMAIN', environ_prefix=None)
```

Comment:
How will this work for local development?

Reply:
What I mean is, will those values be required for local development and if yes, where do I get them? Mind documenting those?

Reply:
I'm unclear on that. This is the documentation I've seen: https://mana.mozilla.org/wiki/display/SECURITY/SSO+Request+Form

Reply:
Hmm, okay, assuming we're still building a somewhat open project we should add some information how to get this running, at least for our team. Maybe we can find the info on the Auth0 page? I'm sure there are also values that we can provide for people that would like to hook it up directly with Google's OIDC endpoint.